msgraph-connector-test/pom.xml

```diff
@@ -58,6 +64,16 @@
       <url>https://oss.sonatype.org/content/repositories/snapshots</url>
     </pluginRepository>
   </pluginRepositories>
+  <repositories>
+    <repository>
+      <id>sonatype</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+      <snapshots>
+        <updatePolicy>always</updatePolicy>
+      </snapshots>
+    </repository>
+  </repositories>
```

Comment:
still needed?

Reply:
yes, the 'dev' workflow run parameterizes the web-tester verison to use ... and passes in a SNAPSHOT :)